components/camel-salesforce/camel-salesforce-maven-plugin/src/main/java/org/apache/camel/maven/GenerateMojo.java

```diff
@@ -85,16 +85,21 @@ public String enumTypeName(final String sObjectName, final String name) {
             return descriptions.externalIdsOf(name);
         }
 
-        public String getEnumConstant(final String value) {
+        public String getEnumConstant(
```

Comment:
I lost a fight with the source checker here. Not sure why it wants a new line after the open parens.